doc/release/upcoming_changes/18535.improvement.rst

```diff
@@ -0,0 +1,8 @@
+Fix power of complex zero
+-------------------------
+``np.power`` now returns the correct result for ``0^{non-zero}``
```

Comment:
Hmm, does this now exceed 80 columns?
```suggestion
``np.power`` now returns a IEEE 754-compliant result for ``0^{non-zero}``
```

Reply:
I wish I had the reference to check that IEEE even says it explicitly (for complex power, not for normal power).  I do know that the C99 bails on `cpow` (at least for warnings), because it is so complicated...